src/NuGet.Core/NuGet.Protocol/Providers/RawSearchResourceV3Provider.cs

```diff
@@ -1,14 +1,15 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using NuGet.Protocol.Core.Types;
+#pragma warning disable 618
 
 namespace NuGet.Protocol
 {
+    [Obsolete]
```

Comment:
@nkolev92 I deleted your other comment for Obsolote attribute by mistake. We have Obsolote attribute at class level and method level for RawSearchResourceV3Provider.cs

Reply:
lgtm

Reply:
Added.